modules/openapi/src/software/amazon/smithy/openapi/fromsmithy/protocols/Smithy4sAbstractRestProtocol.scala

```diff
@@ -280,9 +277,22 @@ abstract class Smithy4sAbstractRestProtocol[T <: Trait]
           .foreach(param.examples)
       }
       val target = context.getModel.expectShape(member.getTarget)
-      val refSchema = context.inlineOrReferenceSchema(member)
```

Comment:
The removal of this is likely the reason why we've lost inlining. 

The change of behaviour related to timestamps should not impact the rest of the code. Moreover, I'm not quite sure why you moved the logic in here.